cumulus/parachains/common/Cargo.toml

```diff
@@ -100,3 +102,7 @@ runtime-benchmarks = [
 	"sp-runtime/runtime-benchmarks",
 	"xcm-executor/runtime-benchmarks",
 ]
+
+# Configure the native runtimes to use.
+rococo-native = ["rococo-runtime-constants"]
+westend-native = ["westend-runtime-constants"]
```

Comment:
I think "native runtime" here is confusing: that usually means the runtime build for native(-architecture) execution, aka not the wasm runtime.

```suggestion
# Test runtimes specific features.
rococo = ["rococo-runtime-constants"]
westend = ["westend-runtime-constants"]
```

Reply:
aha, ok, thank you, changed. I just copied that from here: https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/node/service/Cargo.toml#L186-L188